tests/code_layout/acceptable_missing_doc.py

```diff
@@ -235,7 +235,9 @@
     "QgsScopeLogger": ["QgsScopeLogger(const char *file, const char *func, int line)"],
     "QgsPythonRunner": [
         "evalCommand(QString command, QString &result)=0",
+        "runFileCommand(const QString &filename, const QString &messageOnError=QString())=0",
```

Comment:
better than adding this here would be to add a docstring

https://github.com/qgis/QGIS/pull/60134/files#diff-ec71c7c8b64a4c4e6178b09f34319bc0f3278de4d3e7b5df5b7fa22456898820R71